.ci/gitlab/common.yml

```diff
@@ -1,9 +1,9 @@
 .common:
-  image: ghcr.io/clash-lang/clash-ci-$GHC_VERSION:2022-02-02
+  image: ghcr.io/clash-lang/clash-ci-$GHC_VERSION:2022-05-10
   timeout: 2 hours
   stage: build
   variables:
-    CACHE_FALLBACK_KEY: $CI_JOB_NAME-master-$GHC_VERSION-3
+    CACHE_FALLBACK_KEY: $CI_JOB_NAME-master-$CI_JOB_IMAGE
```

Comment:
This needs the `-3` appended, due to GitLab not adding cache bust tokens to `CACHE_FALLBACK_KEY` :roll_eyes:

Reply:
I'd assumed that I could get rid of this weirdness because it'll be using a new key.
But it's a global thing that stays around and only goes up?

Reply:
yes